Styles/styles.css

```diff
@@ -0,0 +1,111 @@
+h1 {
```

Comment:
These styles will be applied for every h1 in your document. Are you sure you need the same position for all the future headers?